src/plugins/vis_type_timeseries/public/application/components/aggs/calculation.js

```diff
@@ -68,6 +68,7 @@ export function CalculationAgg(props) {
           <EuiSpacer size="xs" />
           <AggSelect
             id={htmlId('aggregation')}
+            timeRangeMode={props.panel.time_range_mode}
```

Comment:
I would like to don't pass `timeRangeMode` in props. Is it possible to use `PanelModelContext` instead?

![image](https://user-images.githubusercontent.com/20072247/128351559-d9767622-e1b2-48b5-ba4d-55bdca67e49c.png)

Reply:
done